XRPLF/xrpl-dev-portal
# rippled v1.1.0 Changes [Waiting for official release]

Combines the following PRs with changes relating to `rippled` 1.1.0:

- #438 (DepositPreauth reference changes)
- #439 (Ubuntu build tutorial updates)
- #443 (SQLite transaction db page size issue and fix)
- #446 (`--unittest-ipv6` commandline option)
- https://github.com/mDuo13/ripple-dev-portal/pull/46 (signing command permission changes)

Also adds a couple minor changes:

- Update Ubuntu instructions to more clearly specify that it requires Boost 1.67.0 exactly (since Boost 1.68.0 is not compatible with rippled 1.1.0)
- Use `master` instead of `develop` for rippled source code links. Update a few line numbers accordingly. (The line numbers for DepositPreauth objects are speculatively set to match what's expected to be merged to master based on the release candidate.)
- Update the `deposit_authorized` method's error codes to match https://github.com/ripple/rippled/pull/2642

This PR needs minimal modifications after the `rippled` 1.1.0 release:

- [ ] Update the build instructions with the version-setting commit for v1.1.0
- [ ] Re-run the link checker to confirm that all the v1.1.0-related links work
